src/main/java/ca/ciccc/Assignment1.java

```diff
@@ -127,7 +159,17 @@ public static String powers(int number) {
      *          Min integer: 5"
      */
     public static String arithmetic(int a, int b) {
-
-        return "";
+        int sum = a + b;
```

Comment:
Yes you need to cast the values of the int to a double to have the correct value